src/content.ts

```diff
@@ -801,17 +801,14 @@ function getYouTubeVideoID(url: string) {
     }
 
     //Get ID from searchParam
-    if (urlObject.searchParams.has("v") && ["/watch", "/watch/"].includes(urlObject.pathname) || urlObject.pathname.startsWith("/tv/watch")) {
+    let m = null;
+    if (urlObject.searchParams.has("v") && urlObject.pathname.match(/^\/((youtube\.com\/)?watch\/?$|tv\/watch)/)) {
```

Comment:
What is `(youtube\.com\/)?` for?

Reply:
youtube-local routes `https://youtube.com/watch?v=<vid>` to `http://your-domain.org/youtube.com/watch?v=<vid>`
It does this because it has to route multiple domains, and it probably makes things simpler.

So if youtube-local were to soon also support another video platform (e.g. dailymotion), and they also use /watch?v=<vid>, this addon will correctly not transmit the id.
Also note that only `^\/watch` would not match youtube-local's routes.